### PR TITLE
feat(connector)_: use status image for DApps connected with Status connector

### PR DESCRIPTION
**Pull request by kounkou**
### Description

Use Status connector badge for DApps connection with Status Connector. Previously only Wallet connect badge was displayed

### What does the PR do

- Added a new image for status connector
- Update the Modal to accept a new parameter
- Keep the change retro-compatible by making parameter not required

### Affected areas

- Request to connect account
- Request for signing transaction

### Architecture compliance

- [x] I am familiar with the application architecture and agreed good practices.
My PR is consistent with this document: [Architecture guidelines](../architecture-guide.md)

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

[Screencast from 2024-08-13 15-35-43.webm](https://github.com/user-attachments/assets/d1bebbe8-b832-471a-805b-f2c840e953e7)

### Impact on end user

User will now see a status icon instead of Wallet connect icon for Dapps connection with Status connector

### How to test

- Launch one instance of Status desktop
- Use a DApps interface to connect with Wallet connect and make sure the icon is the Wallet connect one
- Use a Dapps interface to connect with Status and make sure the icon is the Status connector one

### Risk 

No specific risks

Tick **one**:
- [x] Low risk: 2 devs MUST perform testing as specified above and attach their results as comments to this PR **before** merging.
- [ ] High risk: QA team MUST perform additional testing in the specified affected areas **before** merging.

### CODE REVIEW

**Comment by kounkou** [comment]
Hi, 

To be proactive, this image was generated with below command :

```bash
inkscape -p status-logo-icon.png -o status-connector.svg
```

I reached out to @xAlisher already, and the image can be updated tomorrow as soon as we have the new image.

**Comment by xAlisher** [comment]
Circle logo for Status Connector

**Comment by caybro** [comment]
What happened here? :laughing: 

image

File: ui/StatusQ/src/assets/img/icons/status-connector.svg
Comment (caybro) [inline ui/StatusQ/src/assets/img/icons/status-connector.svg:1]: This is HUGE!

Comment (kounkou) [reply ui/StatusQ/src/assets/img/icons/status-connector.svg:1]: Well... 
Waiting on @xAlisher for the perfect svg 😇

**Comment by jrainville** [comment]
@xAlisher I don,t know what happened with the image, but it's tiny with huge empty margins.
I tried to fix it by modifying the viewbox values, but it gets messed up, so it's probably simpler if you could just try to reupload it.
I think maybe your viewport was super unzoomed when you exported?

**Comment by xAlisher** [comment]
It looks like there was an issue with the export settings. Let me know if this version works better. I also compressed the background to reduce the file size. Additionally, please confirm that all icons are stored on the desktop in 400x400px SVG format.
Circle logo for Status Connector

**Comment by caybro** [comment]
I can take this PR over if you want @kounkou @jrainville

**Comment by jrainville** [comment]
> I can take this PR over if you want @kounkou @jrainville

As you wish, @caybro . If you haven't done it when I start my afternoon, I'll do it

**Comment by caybro** [comment]
> > I can take this PR over if you want @kounkou @jrainville
> 
> As you wish, @caybro . If you haven't done it when I start my afternoon, I'll do it

Ok, go ahead then, plenty stuff in my backlog ☺️

File: ui/StatusQ/src/assets/img/icons/status-connector.svg [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
...T53S4WruTvN3zfE4jTfJXdT2/quuUxQqDiAOf0u5dmPBR9qDFAn+waCrC/NncPeYXd8Gw9efaz2ND89/IIVbiEVUGkUMi0csjxWSXBQW2JMKSScQDGM80zHu4T74dFANVu4BM5EjkkR4hGwdBUyU5BAjWgkIBxWksJIs8+2klfXZ+GET6tBIGOupOmmfI0iAEejEpLIIRFnEeRzMk1POfVtLp5aHxWEdiLNoVZK9DaE2s0gijeJQq0HeHjiE3amf+yHqfNSKhL/8L0+p2lMxDo3A4ivs5hGoOSckfO7MISAoHFMCxCxi+df0nwmM/krp2q/Zrs7BIcwjd5lVziCxCxZb+WDfIl/+R8BijZDdzyZ3T47ro0EvpFtIzndod0mYRduW5SfNq1MUzvZj/pYtiN4itOyNRiPQMsSWH5B1E2hkW8XAdmTfCH9ZLHP9JdwjJbsUad6WxmWSIfncik1CSOaQsEjyiUfAoHDu5479ztJq65KBwsOZZvd8g7a4eoV486CUNCu2Mym5XOFgFo+p/Maxu6+zyINyRG/23Nf8Sd6MISWlmmUdkj1LxKCSX/+1htUCCS+AgG3gc3AFCzTnjQYuGQr1fQWARDYzGoqos8l+7d+zT6dXh8OTOEpV7O6So0AulN2qVxCV4XFz9v3DEKfhio3F1/Rdp6Pk9OHaHhGQQ+aMsQjUWiwi5/O8n+SxMDvMV/e7m0GNv/wa3G0Q4VIuJJYNcXJ79T3GYTs8v6rs/6ztJFtnjkM6DXnfI7n9wzP0LaXL39Limi+N4DTGT6OvCQSs85JD/Y3Ksp8nlQc0f9rtx+dMfZXUWQtL0v02OCZPdX+egWA+ERYRqj4vi4VsUWjL5fwfH9NR1x2y/pqdvi/Mjb7GQXmjMmJxdAuUuGYKG13e9LdJLGHRSERJy44XGnoy/3AnKPi4R6rRShETfQKNZxHGMg+r89IXGXwqUi4O9GeLdv8FNixSSMULoF2v8vel1dXG4mVvDN6pNrKSROGLTHSRZvMD4e1BOscpmzsSleRV7jKxmjxLGeBlT94NydnV5cTi/qYsINJJH/0UWssjAGC8wHsgrwiIW4iEkXoNDQPHiC+mh7cL82SjfqbWXLNo8YaZ4IfHIgsvp2fn51dXl5cXFxc7ywfMjxhIMLi+vrs7Pz06f5Xj6A3DX+cnaEE5XAAAAAElFTkSuQmCC"/>
```
Comment (jrainville) [inline ui/StatusQ/src/assets/img/icons/status-connector.svg:25]: Hmm, this seems weird. I don't think there should be a base 64 image in an svg.

The status-logo SVG for exmaple is way simpler: 
https://github.com/status-im/status-desktop/blob/e279e639a488ce4ca0b8a9cb28f6063d888a92ee/ui/imports/assets/icons/status-logo.svg

@xAlisher can you look again how the SVG was made? It seems like the background is an image

Comment (caybro) [reply ui/StatusQ/src/assets/img/icons/status-connector.svg:25]: The header clearly says image/png do it's just an embedded PNG in an SVG 😑

**Comment by alexjba** [comment]
> @alexjba is this PR still relevant to you or should I just close it? Last update was that the logo I got from Alisher had png background in the SVG so it was unuable.

I totally missed this PR. It's not relevant anymore. Will close it

**Pull request closed without merging**